spec/views/layouts/listnav/_ems_cloud.html.haml_spec.rb

```diff
@@ -10,11 +10,11 @@
   end
 
   it "Flavors link for Openstack cloud manager uses restful path" do
-    record =  ManageIQ::Providers::Openstack::CloudManager.new(:name => "Test Cloud")
+    record = FactoryGirl.create(:ems_openstack, :id => 10)
```

Comment:
Ah, thanks, fixed. Btw. this file should be totally refactored, it's not using Factories for some reason and the tests are not right